CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Changelog
 
+* [CHANGE] server: import godeltaprof/http/pprof
```

Comment:
Can you please specify which endpoints this adds?

Reply:
Done